app/exceptions/exception_handlers.py

```diff
@@ -0,0 +1,21 @@
+from fastapi.responses import JSONResponse
+from fastapi import Request
+from app.exceptions.group_not_found_exception import GroupNotFoundException
+from app.exceptions.user_not_found_exception import UserNotFoundException
+
+
+async def user_not_found_exception_handler(
```

Comment:
Adding too much complexity here.
Just create one module named `exceptions` under the `exceptions` package where you add the two exceptions you created
Then you can raise the exceptions in `service` or in `repository` and create `try...except` blocs for the apis to catch and return the proper response

Reply:
Done!